test/GitIgnoreTest.php

```diff
@@ -29,6 +29,15 @@ public function testIfFileNotExistsItIsCreated()
         $this->assertFileExists($this->gitIgnoreFile);
     }
 
+    public function testIfFileNotExistsItIsNotCreated()
```

Comment:
This name doesn't seem to really describe whats going on, maybe: `testIfGitIgnoreDisabledNoFileIsWritten` ?

Reply:
Haha, I spilled a whole tin of alphabetti spaghetti over this didn't I. 

I'll resolve next week 👍

Reply:
😂 no problem!